test/test-lib-postgresql.sh

```diff
@@ -28,10 +28,6 @@ function test_postgresql_integration() {
 
 # Check the imagestream
 function test_postgresql_imagestream() {
-  case ${OS} in
-    rhel7|centos7|rhel8) ;;
-    *) echo "Imagestream testing not supported for $OS environment." ; return 0 ;;
-  esac
   local tag="-el7"
```

Comment:
Wouldn't the tag be incorrect in case of RHEL9?

Reply:
Fixed by https://github.com/sclorg/postgresql-container/pull/476/commits/d03a6b41c08ac657a57e31897c62858e500b0f9f